AP-protocol.md

```diff
@@ -8,6 +8,14 @@ This document also establishes some technical requirements of the project.
 
 The **Network Initializer** reads a local **Network Initialization File** that encodes the network topology and the drone parameters and, accordingly, spawns the node threads and sets up the Rust channels for communicating between nodes.
 
+## Id Format
+
+ Each ID consists of three fields in the following format: ABN.
```

Comment:
I would advice at using the inverse NBA:
so we have n, 2, 1 bit.

For example if we want to be able to support big number of drone that is possible with this methods without having to do something like.
"ABBNNNNNNNNNNNNNNNNNNNNNN" that would make it very long and need to require a fixed lengthof the N.

Not sure if it is clear.
If we do so than I highly recommend using 1 as the smaller id possible.